.travis.yml

```diff
@@ -24,3 +33,4 @@ matrix:
     allow_failures:
         - env: SYMFONY_VERSION=dev-master
         - php: nightly
+        - php: 7.2
```

Comment:
why allowing failures on 7.2?

Reply:
because it's not released yet. It's RC6, probably stable enough but not released.

Reply:
ok, so idea is to remove it once it is released?

Reply:
Maybe we should add a comment to make it clear

Reply:
yep :slightly_smiling_face: and some changes in this conf should be done when symfony 3.4/4.0 will be released